google-cloud-datastore/src/main/java/com/google/cloud/datastore/Datastore.java

```diff
@@ -168,6 +169,23 @@
    */
   List<Key> allocateId(IncompleteKey... keys);
 
+  /**
+   * Reserve a list of IDs from a complete key
```

Comment:
This doesn't make sense to me, Can you please update this? 

May be something like `Reserve a list of IDs from a complete key` => `Reserve a list of IDs from a multiple keys`

Reply:
done